pkg/pipelines/bootstrap.go

```diff
@@ -20,7 +20,7 @@ import (
 
 var (
 	dockerSecretName = "regcred"
-	saName           = "demo-sa"
+	saName           = "pipeline"
 	roleName         = "tekton-triggers-openshift-demo"
```

Comment:
Please make use of this variable.   Pass it to the create methods so that we only need to change this one variable when we need to rename service account.